group/templates/group/groupDetail.html

```diff
@@ -24,14 +24,23 @@
 {% extends "index/base.html" %}
 {% load static %} 
 {% block body_block %}
+<br>
 <div class="container-fluid">
   <div class="row-fluid">
-    <a href="{% url 'group:list' %}" class="btn btn-sm btn-default" data-toggle="modal" style="float:left">
+    <a href="{% url 'group:list' %}" class="btn btn-sm btn-default" 
+       data-toggle="modal" style="float:left, margin-right:5px">
       return
     </a>
+    <!--Check if user is owner/coowner, if true then display edit button-->
+    {% if isOwner %}
```

Comment:
I suggest that you can use user_is_owner to specify usage of this variable. If so, you don't have to write comment here.